### DISCUSSION TITLE
Any real-life examples of Risor for DevOps?

### DISCUSSION THREAD

**Discussion by mloskot** [deepnoodle-ai/risor#114] [General]
The Risor's description says:

> (...) scripting for Go developers and DevOps. 

Sounds great! But, I'm failing to find any examples that would show case Risor for DevOps, and https://github.com/search?q=repo%3Arisor-io%2Frisor DevOps yields nothing either :)

I'm interested in learning about Risor use cases for driving IaC projects, running Terraform, etc.?

I've seen companies maintain their own tools to juggle configurations, stacks, environments, etc. within their IaC repositories, for example https://github.com/claranet/terraform-wrapper and I myself use https://github.com/magefile/mage for such scripting (as Bash replacement) for exactly same purpose. I bet Risor can do that too, similarly to Python, Bash, etc.

**Comment by myzie** [comment]
You're right about the lack of examples. There are several projects with making use of Risor in these areas, but a couple aren't open source to my knowledge.

My intent is to expand the examples and add more modules that help integrate with K8s, Terraform, etc. These things are happening, but slowly at times, since this is a hobby project for me.

I'd definitely encourage you to consider contributing to Risor if you have the interest and a bit of time here and there!

**Reply by mloskot** [reply]
Thank you for the answer.
Although the idea is quite interesting to me too, I'm still missing the answer to the simple question: Why should I pick Risor?
From DevOps perspective, why should I pick Risor to write my scripting instead of just writing in Go directly.

If I ever come to use Risor and I will be back and possibly with contributions, as I usually do.
